### ISSUE TITLE
NTR: D-glyceric acid high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8327]
New term request D-glyceric acid high in blood (elthat_130404085615):

**Comment by pnrobinson** [comment]
@MickeySegal
Is the "D" important here, or should this be Elevated circulating glyceric acid concentration,

ttps://healthmatters.io/understand-blood-test-results/glyceric-acid

**Comment by MickeySegal** [comment]
https://www.omim.org/entry/220120 "described nonketotic hyperglycinemia with the excretion of D-glyceric acid in the urine and the presence of this substance in the serum".  The clinical synopsis https://www.omim.org/clinicalSynopsis/220120 lists "Increased D-glyceric acid in serum, urine, and CSF"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000371
- label: Elevated circulating D-glyceric concentration
- parents: [Abnormal circulating carboxylic acid concentration (HP:0004354)](https://hpo.jax.org/app/browse/term/HP:0004354)
- pmid: 
- def: The concentration of D-gyceric acid in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8327
- synonyms:

**Comment by MickeySegal** [comment]
Shouldn't the label include the word acid?

It also seems that 2,3-dihydroxypropionic acid is a synonym.

**Comment by pnrobinson** [comment]
check this

**Comment by pnrobinson** [comment]
@MickeySegal 
thanks for pointing this out. Fixed label to "Elevated circulating D-glyceric acid concentration"
